src/Rector/Contrib/PHPUnit/SpecificMethod/AssertSameBoolNullToSpecificMethodRector.php

```diff
@@ -11,24 +11,22 @@
 
 /**
  * Before:
- * - $this->assertSame(null, $anything);
- * - $this->assertSame(true, $anything);
- * - $this->assertSame(false, $anything);
+ * - $this->assertSame({keyword}}, $anything);
+ * - $this->assertNotSame({keyword}}, $anything);
  *
  * After:
- * - $this->assertNull($anything);
- * - $this->assertTrue($anything);
- * - $this->assertFalse($anything);
+ * - $this->assert{keyword}($anything);
+ * - $this->assertNot{keyword}($anything);
```

Comment:
This Rector covers only 6 possible cases, so it would make sense to keep only real examples here, no generic.

E.g.

```
Before:
- $this->assertSame(null, $anything);
- $this->assertNotSame(false, $anything)
```